clio-util/src/main/scala/org/broadinstitute/clio/util/model/UpsertId.scala

```diff
@@ -0,0 +1,6 @@
+package org.broadinstitute.clio.util.model
+
+sealed case class UpsertId(id: String) extends Ordered[UpsertId] {
```

Comment:
I think we want `final` instead of `sealed`.

Reply:
ok